torch/csrc/distributed/rpc/process_group_agent.cpp

```diff
@@ -84,8 +84,8 @@ std::vector<int64_t> ProcessGroupAgent::MessageCounter::snapshot() {
 
 ////////////////////////  ProcessGroupAgent  /////////////////////////////////
 
-constexpr std::chrono::milliseconds INFINITY_SLEEP_INTERVAL =
-    std::chrono::hours(8760);
+const std::chrono::milliseconds INFINITY_SLEEP_INTERVAL =
+    std::chrono::hours(std::numeric_limits<int64_t>::max());
```

Comment:
Looks like there is https://en.cppreference.com/w/cpp/chrono/duration/max that we can use directly.